app/src/main/java/com/pokerio/app/PokerioFirebaseMessagingService.kt

```diff
@@ -23,15 +25,29 @@ class PokerioFirebaseMessagingService : FirebaseMessagingService() {
     }
 
     companion object {
-        fun playerJoined(data: MutableMap<String, String>) {
+        fun playerJoined(data: Map<String, String>) {
             PokerioLogger.debug("Received playerJoined FCM message")
 
             GameState.addPlayer(
                 Player(
-                    data["nickname"] ?: "Player",
-                    data["playerHash"] ?: ""
+                    data["nickname"]!!,
+                    data["playerHash"]!!
                 )
             )
         }
+        fun settingsUpdated(data: MutableMap<String, String>) {
+            PokerioLogger.debug("Received updatedSettings FCM message")
+
+            GameState.changeGameSettings(
+                data["startingFunds"]?.toInt() ?: GameState.startingFunds,
```

Comment:
Please use non-null assertions instead, like this: `data["startingFunds"]!!`

Reply:
Done.